recipes/r-gdalraster/meta.yaml

```diff
@@ -0,0 +1,113 @@
+{% set version = '2.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-gdalraster
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gdalraster_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gdalraster/gdalraster_{{ version }}.tar.gz
+  sha256: 5a52e24e10e44163dfc366007a356dfc27f9d28353d557b507e46d119f3b6ef2
+  patches:
+    - min-ucrt-libs.patch
```

Comment:
```suggestion
    - min-ucrt-libs.patch  # [win]
```

If this patch is only needed for Windows can just apply it as such. Though is fine to keep it generic also.